doc/tutorials/using_windows_as_uos.rst

```diff
@@ -0,0 +1,375 @@
+.. _using_windows_as_uos:
+
+Using Windows as Guest OS on ACRN
+#################################
+This tutorial describes how to launch Windows as the Guest OS on the ACRN hypervisor.
+
+Hardware setup
+**************
+The following Intel Kaby Lake NUCs are verified:
+
+.. csv-table::
+   :header: "Platform Model", "Bios Version", "Download Link"
+
+   "NUC7i7DNHE", "DNKBLi7v.86A.0052.2018.0808.1344", "`link <https://downloadcenter.intel.com/download/28886?v=t>`__"
+   "NUC7i5DNHE", "DNKBLi5v.86A.0060.2018.1220.1536", "`link <https://downloadcenter.intel.com/download/28885?v=t>`__"
+
+ACRN Service OS Setup
+*********************
+You may refer to the steps in :ref:`getting-started-apl-nuc` for
+Intel NUC to setup ACRN on KBL NUC. After following the steps in that guide,
```

Comment:
Intel NUC to setup ACRN on KBL NUC.


Change to:
Intel NUC to set up ACRN on the KBL NUC